data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,513 @@
+			`	After waiting for what seems like an eternity you spot two hooded figures approaching you, whispering amongst themselves. You start to doubt whether or not this was a good idea, but it is probably too late to back down now. They come to address you, their face somehow impossible to glimpse at, even below the hoodie. "Hello, <last>," they say at the same time, almost in a comical way, removing some of the tension you felt until now. Then they look at each other, nod, and one of them addresses you in a more serious tone. "We are sorry for all of these complications, but the truth is, we cannot afford to take risks. You see, we simply do not know how you might react. Furthermore, it is best for all if you do not come to know us."`
+			choice
+				`	"Tell me more."`
+					goto tellme
+				`	"How do you know me?"`
+					goto know
+				`	(Run away).`
+			`	They seem surprised at your sudden change of mind, but make no effort to stop you.`
+			label trainback
+			`	You proceed to head to the nearest spaceport by getting yourself another train ticket, without any incidents.`
+				decline
+			label know
+			`	They grin, "You need to become known by the Hai in order to land on our homeworld."`
+			`	"We know you have provided help, by doing various jobs."`
+				to display
+					"reputation: Hai (Unfettered)" >= -1250
+			`	"Do not think we are unaware of your skirmishes with the Unfettered," they add with a serious tone.`
+				to display
+					"reputation: Hai (Unfettered)" < -1250
+			label tellme
+			`	Suddenly, you hear a loud roar behind you as the ground begins to shake.`
+			`	Turning back towards the sea where the noise is coming from you see the waves getting more agitated and the wind rise as a shape emerges out of the water surrounded by fumes, as if an ancient sea monster was arising from the depths.`
+			`	But it is no monster. Indeed it is so, so much worse.`
+			`	It is a ship, and its guns are pointed straight at you!`
+			label tryagain
+			scene "scene/emerging ladybug"
+			choice
+				`	(Run!)`
+				`	(Wait.)`
+					goto wait
+				`	(Try to contact my fleet for help!)`
+				`	(Shoot it!)`
+					goto dumb
+			`	As you try to react to this sudden threat, the Hai closest to you firmly grabs your arm and shouts to get trough the noise. "Wait! If this ship wanted to kill you, <last>, you would already be dead."`
+			label wait
+			`	You are having a hard time hearing anything with the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm.`
+			`	They turn to you, surprisingly calm in this situation. "Lucky for you, this is our ship. But not any ship! This is a mighty relic of the past, from the times our Empire used to control the stars: a Ladybug!`
+			choice
+				`	(Say nothing.)`
+					goto explain
+				`	"You could have told me before."`
+			`	"Yes, but where is the fun in that? We also wanted to see how you would react.`
```

Comment:
I love this pair of squirrels :D

Reply:
They are truly Unfettered in their heart x)